.gitignore

```diff
@@ -0,0 +1,3 @@
+/node_modules
+package-lock.json
+/client/node_modules
```

Comment:
Consider add more patterns for examples for ide's specific files - visual code studio etc